JASP-Desktop/mainwindow.cpp

```diff
@@ -474,6 +486,12 @@ void MainWindow::loadQML()
 	_upgrader->loadOldSchoolUpgrades();
 }
 
+void MainWindow::showRCommander()
+{
+	Log::log() << "Loading RCmdWindow"  << std::endl;
+	_qml->load(QUrl("qrc:///components/JASP/Widgets/RCommanderWindow.qml"));
+}
+
```

Comment:
Why should we need several R windows?

Reply:
Why not?

Reply:
My $0.02:
I wanted to retrieve the R popup after it went behind some window and did it by clicking on the menu item again. Which promptly opened a new popup window -- behaviour I was not expecting. Additionally, during all my use of RStudio I've only ever had two instances of an R session open if I was working in different RStudio projects, which we do not support. So I'd be in support of a single instance.

Reply:
I agree, until JASP I've never had a reason for multiple R windows.

Reply:
Ok!
But what should the behaviour of the R-ribbonbutton be then?
Click it first time -> Open window
Click it second time? Show window? Close window?

Reply:
Clicking it a second time should make it the active window and if it already is, then it should do nothing